envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyAdmin.kt

```diff
@@ -44,7 +44,11 @@ class EnvoyAdmin(
             }
 
     fun statValue(statName: String): String? = get("stats?filter=$statName").body()?.use {
-        it.string().lines().first().split(":").get(1).trim()
+        val splitedStats = it.string().lines().first().split(":")
+        if (splitedStats.size != 2) {
+            return "0"
```

Comment:
Are we sure? Some tests rely on 0 value, maybe -1?

Reply:
Done